x-pack/plugins/actions/server/builtin_action_types/jira/types.ts

```diff
@@ -207,7 +186,7 @@ export interface GetIssueHandlerArgs {
 
 export interface ExternalServiceApi {
   getFields: (args: GetCommonFieldsHandlerArgs) => Promise<GetCommonFieldsResponse>;
-  getIncident: (args: GetIncidentApiHandlerArgs) => Promise<void>;
+  getIncident: (id: GetIncidentApiHandlerArgs) => Promise<ExternalServiceParams | undefined>;
```

Comment:
Based on `GetIncidentApiHandlerArgs` type, `args` is an object. I think renaming to `id` is confusing.